docs/extensions/jvm.md

```diff
@@ -101,3 +101,31 @@ export interface JvmRunEnvironmentResult{
     environmentVariables: Map<String, String>;
 }
 ```
+
+### JVM Dependency Module
+
+`JvmDependencyModule` is a basic data structure that contains jvm-specific
+metadata. This metadata is embedded in the `data: Option[Json]` field of the `DependencyModule` definition, when the `dataKind` field contains "jvm".
+
+```ts
+export interface JvmDependencyModule {
+  organization: String;
+  name: String;
+  version: String;
+
```

Comment:
If it is a `Maven` dependency then it should also contain a optional `scope?: String`
```suggestion
scope?: String

```
Most of the time `scope` can be left empty, meaning it is a dependency on the `"compile"` scope.

Reply:
Thx! Added